### DISCUSSION TITLE
Feets module ?

### DISCUSSION THREAD

**Discussion by AIrtistry** [Bing-su/adetailer#275] [Ideas]
Think add a feet module can be a great idea, asd hands, it's pretty hard to have decent results on a large scale of images.

**Comment by Bohdan1321** [comment]
Yes

**Comment by OLya124** [comment]
Yes!

**Comment by Symbiomatrix** [comment]
Sure, but do you have one that's been trained in mind?

**Comment by left1000** [comment]
Almost all my ai results have 11 or 12 toes (if they have feet) :(
